CONTRIBUTING.md

```diff
@@ -93,6 +93,12 @@ The process of proposing a change to Talawa Admin can be summarized as:
          npm install --legacy-peer-deps
          npm run test --watchAll=false --coverage
       ```
+      
+      2. Debug Tests in browser 
+      ```
+         npm install --legacy-peer-deps
+         npm run jest-preview
+         npm run test --watchAll=false --coverage
```

Comment:
1. What is the URL that should be used?
1. What will the user expect to see when tests fail? Can you provide images of both cases?

Reply:
Yes sure